interface_mock.go

```diff
@@ -0,0 +1,105 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: interface.go
+
+// Package helmclient is a generated GoMock package.
+package helmclient
```

Comment:
Maybe put the mock into a dedicated subpackage, so it doesn't "pollute" the main package? `helmclient/mock` with `package mockhelmclient` or similar?

Reply:
I'm with you on that, but this would result in an import cycle:
```
package github.com/mittwald/go-helm-client
	imports github.com/mittwald/go-helm-client/mock
	imports github.com/mittwald/go-helm-client
```

As `client_test.go` only tests the mocked interface methods from `interface_mock.go`, moving the former to the same `mockhelmclient`-subpackage would seem to be the most sensible way. Your thoughts on this?

Reply:
Sounds good. As mentioned in the other comment, the unit test testing the mock is not really necessary anyhow, IMHO. Moving it into the `mockhelmclient` package or dropping it entirely would both do the job.